routes/quoteCalc.js

```diff
@@ -1,9 +1,28 @@
 const express = require("express");
 const router = express.Router();
 
-//Below is for get, change get to post for post request
-router.get("api url", (req, res) => {
-  //Api code
+function calculatePremium(car_value, risk_rating) {
+  const yearly_premium = parseFloat(
+    ((car_value * risk_rating) / 100).toFixed(2)
+  );
+  const monthly_premium = parseFloat((yearly_premium / 12).toFixed(2));
+  return { monthly_premium, yearly_premium };
+}
+
+router.post("/get_quote", (req, res) => {
+  console.log("Received request:", req.body); 
+  const { car_value, risk_rating } = req.body;
+
+  if (typeof car_value !== "number" || car_value <= 0) {
```

Comment:
Validation inside the route makes maintenance difficult

Reply:
Great point, thank you - validation has been separated from the routing logic.